cropper/src/main/java/com/canhub/cropper/CropOverlayView.kt

```diff
@@ -908,28 +908,28 @@ class CropOverlayView
     ) {
         // Top left
         canvas.drawCircle(
-            rect.left,
+            rect.left - cornerExtension,
             (rect.top - cornerExtension),
```

Comment:
Added this change to align properly the corners over the edges of the rectangle. Observed this while checking it for smaller radius.